README.md

```diff
@@ -1,75 +1,66 @@
+![Travis CI badge](https://travis-ci.com/rdupplaw/bowling-challenge.svg?branch=master)
 
-Bowling Challenge
-=================
+# Bowling Challenge in JavaScript
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+## About
 
-## The Task
+This is a JavaScript rewrite and slight redesign of [my Bowling Challenge in Ruby](https://github.com/rdupplaw/bowling-challenge-ruby).
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+This program calculates the score of a bowling game given the rolls. A user can input the rolls sequentially as they occur:
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+```javascript
+game.bowl(5) // => 5
+game.bowl(3) // => 8
+// etc.
+```
 
-A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
+The `Game.prototype.bowl()` method returns the current total score, following [traditional ten-pin bowling rules](https://en.wikipedia.org/wiki/Ten-pin_bowling#Traditional_scoring). There is no need to enter `game.bowl(0)` after a strike; the game logic will automatically move onto a new frame when a strike occurs.
 
-As usual please start by
+The game does not validate the pinfall of rolls, so there's nothing to stop a user from inputting `game.bowl(100)`. The game does not stop you from bowling after the last frame, but the scoring does not take into account any frames after the last frame, except for bonuses.
 
-* Forking this repo
+### Design
```

Comment:
Really clear readme and very informative - good job for explaining the design and how your classes follow SRP!